data.sql

```diff
@@ -32,4 +32,39 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
-UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+
+
+
+--## Vet clinic database: add "join table" for visits
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '04-23-2000');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '01-17-2019');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '05-04-1981');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, '06-08-2008');
+
+INSERT INTO specializations (vets_id, species_id) VALUES (1, 1);
```

Comment:
[Optional]
- [ ]  Directly using foreign key ids is not practical in real-life projects. It will be better if you query id with the name 
```SQL
(SELECT id FROM vets WHERE name = 'William Tatcher')
```

Reply:
Thanks for the approval and the recommendation!